exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,5 @@
-const name = " Daniel  ";
+const names = "  Daniel  ";
+const message = names.trim();
+// const nameLength = names.length;
 
-console.log(message);
+console.log(nameLength);
```

Comment:
As with the previous exercise, you need to use .length to get the number of letters in the name